From: Javier Viguera <javier.viguera@digi.com>
Date: Thu, 4 May 2017 17:03:37 +0200
Subject: [PATCH] ublox: support modems connected to serial port

Add the vendor string, so it can be probed via AT commands. This allows
to support modems that are connected to a serial port.

Upstream-Status: Pending

Signed-off-by: Javier Viguera <javier.viguera@digi.com>
---
 plugins/ublox/mm-plugin-ublox.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/plugins/ublox/mm-plugin-ublox.c b/plugins/ublox/mm-plugin-ublox.c
index 890f520754aa..0ca74d903f88 100644
--- a/plugins/ublox/mm-plugin-ublox.c
+++ b/plugins/ublox/mm-plugin-ublox.c
@@ -66,11 +66,13 @@ mm_plugin_create (void)
 {
     static const gchar *subsystems[] = { "tty", "net", NULL };
     static const guint16 vendor_ids[] = { 0x1546, 0 };
+    static const gchar *vendor_strings[] = { "u-blox", NULL };
 
     return MM_PLUGIN (g_object_new (MM_TYPE_PLUGIN_UBLOX,
                                     MM_PLUGIN_NAME,               "u-blox",
                                     MM_PLUGIN_ALLOWED_SUBSYSTEMS, subsystems,
                                     MM_PLUGIN_ALLOWED_VENDOR_IDS, vendor_ids,
+                                    MM_PLUGIN_ALLOWED_VENDOR_STRINGS, vendor_strings,
                                     MM_PLUGIN_ALLOWED_AT,         TRUE,
                                     MM_PLUGIN_CUSTOM_AT_PROBE,    custom_at_probe,
                                     NULL));
